src/box/lua/config/applier/console.lua

```diff
@@ -3,6 +3,8 @@ local console = require('console')
 local log = require('internal.config.utils.log')
```

Comment:
I think that we should add a changelog entry, because the patch fixes a user visible behavior. The change is not internal: `NO_CHANGELOG=internal` is incorrect.

Also, the `NO_DOC` motivation can't be 'internal' too. I would just say that it is a bugfix.

Reply:
corrected